src/Oscoin/Storage/Block/Pure.hs

```diff
@@ -100,6 +102,14 @@ getBlocks (fromIntegral -> d) Handle{..} =
 -- Nb. we guarantee that there is at least one chain in the store by exposing
 -- only the 'genesisHandle' smart constructor.
 
+getChainSuffix
+    :: (Ord (BlockHash c))
+    => BlockHash c
+    -> Handle c tx s
+    -> [Block c tx (Sealed c s)]
```

Comment:
Could you add some documentation here? The function would probably also benefit from `Oscoin.Time.Chronob.